app/controllers/users_controller.rb

```diff
@@ -5,6 +5,6 @@ def index
 
   def show
     @user = User.find(params[:id])
-    @posts = @user.posts
+    @posts = @user.recent_posts
```

Comment:
- [ ] Please display only `3 recent posts` on the user posts page and remember to add this button `<%= link_to 'See all Post', user_posts_path(@user) %>` that lead you where you can see all posts

Reply:
Fixed.